iot_input_oca/models/iot_device_input.py

```diff
@@ -113,7 +113,7 @@ class IoTDeviceAction(models.Model):
     _name = "iot.device.input.action"
     _description = "Action of device inputs"
 
-    input_id = fields.Many2one("iot.device.input")
+    input_id = fields.Many2one("iot.device.input", ondelete="cascade")
```

Comment:
Is this the history of the communications? Should be removed?

Reply:
Well, it is the log, if you are removing the input, it might be necessary,  Also, it might have sense to purge them, as it might not have sense to store all history (maybe a configuration by device or input/output)

Reply:
Can be this log very verbose? I have seen 477K entries for a device with only few months of life.

Reply:
Yes, it has sense, let's make it cleaner!